src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -78,7 +78,11 @@ export interface IEditorGroupsControl {
 	setGroupOrientation(orientation: GroupOrientation): void;
 	getGroupOrientation(): GroupOrientation;
 
+	requestActiveGroupSizeChange(groupSizeChange: number): boolean;
```

Comment:
Can we maybe call this similar to the other one `resizePart` in `partService`, e.g. `resizeGroup` and you pass in the position of the group to resize?